frontend/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -546,24 +547,30 @@ const DashboardPage = ({ router, location }: IDashboardProps): JSX.Element => {
     ),
   });
 
-  const MDMCard = useInfoCard({
-    title: "Mobile device management (MDM)",
-    titleDetail: mdmTitleDetail,
-    showTitle: !isMacAdminsFetching,
-    description: (
-      <p>MDM is used to change settings and install software on your hosts.</p>
-    ),
-    children: (
-      <Mdm
-        isFetching={isMdmFetching}
-        error={errorMdm}
-        mdmStatusData={mdmStatusData}
-        mdmSolutions={mdmSolutions}
-        selectedPlatformLabelId={selectedPlatformLabelId}
-        selectedTeamId={currentTeamId}
-      />
-    ),
-  });
+  const MDMCard = (
+    <SandboxGate>
+      {useInfoCard({
```

Comment:
For readability, id suggests we keep the `MdmCard` variable and instead of removing it and calling `useInfoCard` inline here.

```tsx
<SandboxGate>
  {MdmCard}
</SandboxGate?
```